src/machines/activity_state_machine.rs

```diff
@@ -163,7 +323,10 @@ impl StartedActivityCancelCommandCreated {
 pub(super) struct StartedActivityCancelEventRecorded {}
 
 impl StartedActivityCancelEventRecorded {
-    pub(super) fn on_activity_task_completed(self) -> ActivityMachineTransition {
+    pub(super) fn on_activity_task_completed(
+        self,
+        attrs: ActivityTaskCompletedEventAttributes,
+    ) -> ActivityMachineTransition {
         // notify_completed
         ActivityMachineTransition::default::<Completed>()
```

Comment:
I believe this needs to issue the `Complete` command as well, given in Java the same thing  happens here as from `Started`->`Completed`

Reply:
Yep, it should be the same thing as in the Started->Completed.

Reply:
This is resolved but the code isn't changed -- not sure if you meant to do that?

Reply:
Sorry, that's my (probably bad) habit of replying right away and pushing changes later.